src/components/PathStep/styled/index.js

```diff
@@ -20,20 +26,21 @@ export const StyledStepItem = attachThemeAttrs(styled.li)`
     cursor: pointer;
     font-size: ${FONT_SIZE_TEXT_LARGE};
     font-weight: 400;
+    transition: all 0.3s linear;
 
     &:hover {
```

Comment:
@wildergd add the &: fucus and the &: activate so that it does not put the one on the system.


![Screenshot from 2020-12-08 11-32-22](https://user-images.githubusercontent.com/332409/101513284-ff169d00-3949-11eb-9baf-2e4af3d0b297.png)